pkgs/by-name/at/atracdenc/package.nix

```
@@ -0,0 +1,35 @@
+{
+  cmake,
+  fetchFromGitHub,
+  lib,
+  libsndfile,
+  stdenv,
+}:
+stdenv.mkDerivation rec {
```

this rec is unnecessary
```suggestion
stdenv.mkDerivation {
```

---

Thank you, removed this `rec`